### ISSUE TITLE
Remove extensions

### ISSUE THREAD

**Issue by rEnr3n** [xsmile/chromexup#3]
How should I properly remove extensions? I removed one of my extensions from the browser settings. I initiated chromexup to install it again but it's not showing up. I also tried deleting the files in `External Extensions` folder. Same thing.

**Comment by xsmile** [comment]
By removing the extension manually it was [blacklisted](https://developer.chrome.com/apps/external_extensions#faq) (see the bottom of the page). To remove it from the blacklist, you need to install it by dropping the crx file in the `chrome:extensions` page and to uninstall it again. I should probably mention it in the description.

chromexup can remove extensions if you enable the `remove_orphans` setting in the configuration file.
Extensions not listed in the `[extensions]` section will be removed the next time the script is started.

To sum it up, use only the configuration file to add or remove extensions.

**Comment by xsmile** [comment]
The FAQ section covers this question now.
